client/rhel/spacewalk-client-tools/src/up2date_client/rhnreg_constants.py

```diff
@@ -193,62 +191,62 @@
                             "grouping, systems permissions, system package profiling, "
                             "bare-metal provisioning, existing state provisioning, "
                             "rollbacks, configuration management")
 
-VIRT = _("Virtualization module: software updates for a limited number of "
+VIRT = _("Virtualization type: software updates for a limited number of "
         "virtual guests on this system.")
 
 
 VIRT_FAILED = _("<b>Warning:</b> Any guest systems you create on this system "
-        "and register to RHN will consume Red Hat Enterprise Linux "
+        "and register to Uyuni will consume system "
         "subscriptions beyond this host system's subscription. You will need "
         "to: (1) make a virtualization system entitlement available and "
         "(2) apply that system entitlement to this "
-        "system in RHN's web interface if you do not want virtual guests of "
+        "system in Uyuni web interface if you do not want virtual guests of "
         "this system to consume additional subscriptions.")
 
 NO_SYS_ENTITLEMENT         = _("This system was unable to be associated with "
-"any RHN service level modules. This system will not receive any software "
-"updates to keep it secure and supported. Contace your Satellite administrator "
+"any Uyuni system types. This system will not receive any software "
+"updates to keep it secure and supported. Contace your Uyuni administrator "
 "about this problem. Once you make the "
 "appropriate active subscriptions available in your account, you may browse "
-"to this system's profile in the RHN web interface, delete the profile, and "
-"re-connect this system to Red Hat Satellite.")
+"to this system's profile in the Uyuni web interface, delete the profile, and "
+"re-connect this system to Uyuni.")
 ACTIVATION_KEY          = _("Universal default activation key detected\n"
                             "A universal default activation key was detected in your account. "
                             "This means that a set of properties (software channel subscriptions, "
                             "package installations, system group memberships, etc.) "
-                            "for your system's connection to Red Hat Satellite or Red Hat Satellite Proxy"
+                            "for your system's connection to Uyuni Server or Uyuni Proxy"
                             "have been determined by the activation key rather than your "
                             "installation number.  "
-                            "You may also refer to the RHN Reference Guide, section 6.4.6 for more details "
-                            "about activation keys (http://access.redhat.com/knowledge/docs/Red_Hat_Network/)\n"
+                            "You may also refer to the Uyuni Reference Guide for more details "
+                            "about activation keys (https://www.uyuni-project.org/uyuni-docs/uyuni/reference/intro.html)\n"
```

Comment:
This will definitely look weird to downstream users, referring them to Uyuni docs instead of SUSE Manager docs :-/